rct229/ruleset_functions/baseline_systems/is_baseline_system_10.py

```diff
@@ -0,0 +1,137 @@
+    are_all_terminal_types_cav_with_none_equal_to_null,
+)
+from rct229.ruleset_functions.baseline_systems.baseline_hvac_sub_functions.do_all_terminals_have_one_fan import (
+    do_all_terminals_have_one_fan,
+)
+from rct229.ruleset_functions.baseline_systems.baseline_hvac_sub_functions.does_each_zone_have_only_one_terminal import (
+    does_each_zone_have_only_one_terminal,
+)
+from rct229.ruleset_functions.baseline_systems.baseline_hvac_sub_functions.does_hvac_system_serve_single_zone import (
+    does_hvac_system_serve_single_zone,
+)
+from rct229.ruleset_functions.baseline_systems.baseline_hvac_sub_functions.is_hvac_sys_cooling_type_none import (
+    is_hvac_sys_cooling_type_none,
+)
+from rct229.ruleset_functions.baseline_systems.baseline_hvac_sub_functions.is_hvac_sys_fan_sys_CV import (
+    is_hvac_sys_fan_sys_cv,
+)
+from rct229.ruleset_functions.baseline_systems.baseline_hvac_sub_functions.is_hvac_sys_heating_type_elec_resistance import (
+    is_hvac_sys_heating_type_elec_resistance,
+)
+from rct229.ruleset_functions.baseline_systems.baseline_system_util import (
+    HVAC_SYS,
+    find_exactly_one_hvac_system,
+)
+
+HEATING_SYSTEM = schema_enums["HeatingSystemOptions"]
+
+
+def is_baseline_system_10(rmi_b, hvac_b_id, terminal_unit_id_list, zone_id_list):
+    """
+     Get either Sys-10 or Not_Sys_10 string output which indicates whether the HVAC system is ASHRAE 90.1 2019 Appendix G system 10 (Heating and Ventilation with electric heating).
+
+     Parameters
+     ----------
+     rmi_b json
+         To evaluate if the hvac system is modeled as either Sys-10 or Not_Sys_10 in the B_RMI.
+     hvac_b_id list
+         The id of the hvac system to evaluate.
+     terminal_unit_id_list
+         List list of terminal unit IDs associated with the HVAC system to be evaluated. These are sent to this function from the master get_baseline_system_types function.
+     zone_id_list list
+         list of zone IDs associated with the HVAC system to be evaluated. These are sent to this function from the master get_baseline_system_types function.
+
+    Returns
+     -------
+     The function returns either Sys-10 or Not_Sys_10 string output which indicates whether the HVAC system is ASHRAE 90.1 2019 Appendix G system 10 (Heating and Ventilation with electric heating).
+    """
+
+    is_baseline_system_10 = HVAC_SYS.UNMATCHED
+
+    hvac_b = find_exactly_one_hvac_system(rmi_b, hvac_b_id)
+
+    # check if the hvac system has the required sub systems for system type 10
+    preheat_system = hvac_b.get("preheat_system")
+    has_required_preheat_sys = (
+        preheat_system is None
+        or preheat_system.get("heating_system_type") is None
+        or preheat_system["heating_system_type"] == HEATING_SYSTEM.NONE
+    )
+
+    heating_system = hvac_b.get("heating_system")
+    has_required_heating_sys = (
```

Comment:
The check returns True if no required heating sys. Seems we have rename it to:

has_heating_sys?
I suggest use Not instead of repeating the same logic twice.